comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -47,8 +47,10 @@
       <packagereq type="default">ruby193-rubygem-puppetdb_foreman</packagereq>
       <packagereq type="default">rubygem-foreman_scap_client</packagereq>
       <packagereq type="default">rubygem-newt</packagereq>
+      <packagereq type="default">rubygem-sequel</packagereq>
```

Comment:
rubygem-newt isn't a core package either, not sure I understand the sections in comps here